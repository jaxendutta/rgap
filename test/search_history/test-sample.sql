SELECT * FROM SearchHistory WHERE user_id = 5 ORDER BY search_time DESC;

DELETE FROM SearchHistory WHERE history_id = 24;

SELECT * FROM SearchHistory WHERE user_id = 5 ORDER BY search_time DESC;
