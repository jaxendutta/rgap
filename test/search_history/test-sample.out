+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+
|history_id| user_id |search_recipient | search_grant | search_institution     | search_filters | search_time         | result_count | saved |
+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+
|       23 |       5 | NULL            | scholarship  | University of Waterloo |           NULL | 2025-02-14 23:50:49 |           10 | false |
+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+
|       24 |       5 | NULL            | masters      | University of Waterloo |           NULL | 2025-02-14 23:55:49 |           5  | true  |
+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+
|       25 |       5 | NULL            | research     | University of Waterloo |           NULL | 2025-02-14 23:57:49 |           3  | false |
+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+

Query OK, 1 row affected (0.10 sec)

+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+
|history_id| user_id |search_recipient | search_grant | search_institution     | search_filters | search_time         | result_count | saved |
+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+
|       23 |       5 | NULL            | scholarship  | University of Waterloo |           NULL | 2025-02-14 23:50:49 |           10 | false |
+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+
|       25 |       5 | NULL            | research     | University of Waterloo |           NULL | 2025-02-14 23:57:49 |           3  | false |
+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+

